README.md

```diff
@@ -1,24 +1,91 @@
-# README
+![](https://img.shields.io/badge/Microverse-blueviolet)
 
-This README would normally document whatever steps are necessary to get the
-application up and running.
+# Rails-cook-book
 
-Things you may want to cover:
+The Rails cook book  app keeps track of all your recipes, ingredients, and inventory. It will allow you to save ingredients, keep track of what you have, create recipes, and generate a shopping list based on what you have and what you are missing from a recipe. Also, since sharing recipes is an important part of cooking the app should allow you to make them public so anyone can access them.
 
-* Ruby version
+# Getting Started
 
-* System dependencies
 
-* Configuration
+_To get a local copy up and running follow these simple steps._
 
-* Database creation
+1. Clone the repo
+   ```sh
+   git clone https://github.com/your_username_/Project-Name.git
+   ```
+2. Goto project directory
+   ```sh
+   cd recipe-app
+   ```
 
-* Database initialization
+3. Configure ` change  the Env credentials with your username and password
+4. Run app
+   ```sh
+   rails server
+   ```
+   or
+   ```sh
+   rails s
+   ```
 
-* How to run the test suite
 
-* Services (job queues, cache servers, search engines, etc.)
+# Testing
 
-* Deployment instructions
+`bundle exec rspec` will run all the tests.
 
-* ...
+Also, you can run `bundle exec rspec spec/` to run specific tests.
+
+You can also run `RAILS_ENV=test rspec spec/` to run the tests in test mode.
+
+## Errors
+
+If you encounter any errors, run the following commands.
+
+- Run `rails db:drop db:create db:migrate` to drop, create and migrate a new database.
+
+- Run `rubocop && rubocop -A` to check for and fix code errors.
+
+# This project was built with
+
+- Ruby on Rails
+
+- PostgreSQL
+
+- Bootstrap CSS Framework
+
+# Author 1
+
+👤 **Ben**
+
+- GitHub: [Ben Kiarie](https://github.com/Benmuiruri)
+- Twitter: [Optimize](https://twitter.com/_optimize)
+- LinkedIn: [Ben Kiarie](https://www.linkedin.com/in/benjamin-kiarie-180b66149/)
+
+
+# Author 2
+
+👤 **Ezekiel Utshudi**
+- GitHub: [@EzekielUtshudi](https://github.com/EzekielUtshudi)
+- Twitter: [@UtshudiEzekiel](https://twitter.com/UtshudiEzekiel)
+- LinkedIn: [@ezekiel-utshudi](https://www.linkedin.com/in/ezekiel-utshudi-195782162/)
```

Comment:
- [ ] There is no live link for your project, also there is no link to the video description of your project, Kindly fix this

Reply:
Hi @VICTOR-CHUKWU I think you have this mistaken for a capstone project.

There was no mention of a video or deployment.

Please check the requirements here https://github.com/microverseinc/curriculum-rails/blob/main/recipe-app/buisness_requirements.md